cfgov/unprocessed/css/organisms/video-player.less

```
@@ -48,18 +53,52 @@
 
 .o-video-player_image-container {
     position: relative;
-    // @black not used because it's not totally black.
-    background-color: #000;
+    background-color: #fff;
     text-align: center;
     height: 100%;
-    width: 100%;
+    display: flex;
+
+    // Create an aspect ratio box, see https://css-tricks.com/aspect-ratio-boxes/#the-psuedo-element-tactic
+    &:before {
+        content: '';
+        width: 1px;
+        margin-left: -1px;
+        float: left;
+        height: 0;
+        // 1280px / 720px (16/9) aspect ratio.
+        padding-top: 720px / 1280px * 100%;
```

No need for the pixel values, we can just make this abstract.

```suggestion
        // Calculate padding to set the height of the container proportionally
        // so that we end up with a 16:9 aspect ratio.
        padding-top: 9 / 16 * 100%;
```